### ISSUE TITLE
Per-entity min/max attribute not working

### ISSUE THREAD

**Issue by ofalvai** [custom-cards/bar-card#7]
I'm using bar-card with an entity list, where each entity has a different `min` and `max` attribute. Relevant pars of my config:

```
type: 'custom:bar-card'
columns: 4
entities:
  - entity: sensor.bme280_sensor_humidity
    title: Humidity
    min: 0
    max: 100
  - entity: sensor.pm2_5
    title: PM2.5
    min: 0
    max: 30
  - entity: sensor.pm10
    title: PM10
    min: 0
    max: 30
  - entity: sensor.waqi_pm10
    title: PM10 outside
    min: 0
    max: 30
```

However, it looks like every bar is rendered with a range of 0-100:

![image](https://user-images.githubusercontent.com/1694986/57966107-0d5b2d80-794e-11e9-83ad-9c16af9741fb.png)

What's strange is that if I define a `max` attribute on the main object, all bars are rendered with respect to their own `min` and `max` attributes:

![image](https://user-images.githubusercontent.com/1694986/57966402-af304980-7951-11e9-9828-7c9ea261510b.png)

**Comment by Gluwc** [comment]
Should be fixed in 1.2.0

**Comment by ofalvai** [comment]
Great, thanks for the fix!
